### DISCUSSION TITLE
Welcome to openebs Discussions!

### DISCUSSION THREAD

**Discussion by kmova** [openebs/openebs#3112] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by orville-wright** [comment]
We will be using GitHub discussions more frequently in our newly updated community mode, as to align better with CNCF guidelines.
